client/src/components/my-mentor-card/use-with-recommended-action.tsx

```diff
@@ -145,16 +145,15 @@ export function UseWithRecommendedAction(
   );
   const idleIncomplete = idle?.status === Status.INCOMPLETE;
   const isVideo = mentor?.mentorType === MentorType.VIDEO;
-  const hasThumbnail = mentor?.thumbnail != "";
-
+  const hasThumbnail = /(ftp|http|https):/.test(mentor?.thumbnail);
```

Comment:
same q: not crazy about this scheme test as opposed to just testing if the string is non-empty...unless we really need it this way?